source/_sass/page.scss

```diff
@@ -1,333 +1,330 @@
 body {
-	background-color: $bodycolor;
-	font-family: $base-font;
-	color: $text-color;
+  background-color: $bodycolor;
+  font-family: $base-font;
+  color: $text-color;
 }
 .navigation-wrapper {
-	@include container;
-	padding: 2em 0 1em;
-	font-family: $heading-font;
-	font-weight: 700;
-	text-transform: uppercase;
-	@include clearfix;
+  @include container;
+  @include clearfix;
+  padding: 40px 0;
+  font-family: $heading-font;
+  height: 150px;
+  box-sizing: border-box;
+  border-bottom: 1px solid #ebebeb;
 }
 .site-name {
-	@include grid(12,10);
-	@include prefix(12,1);
-	@include suffix(12,1);
-	margin-bottom: 1em;
-	float: none;
-	display: block;
-	@include font-rem(24);
-	@media #{$small} {
-		@include grid(12,2);
-		@include prefix(12,0.5);
-		@include suffix(12,0.5);
-		@include font-rem(16);
-	}
-	@media #{$x-large} {
-		@include grid(12,1.5);
-		@include prefix(12,2);
-	}
+  @include grid(12,10);
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Commas in mixin arguments should be followed by a single space